spec/FrameSpec.js

```diff
@@ -0,0 +1,158 @@
+describe("Frame", function() {
+  var frame;
+
+  beforeEach(function() {
+    frame = new Frame();
+  });
+
+  describe('playing', function(){
+
+    it("registers a gutter ball", function() {
+      frame.registerGo(0);
+      expect(frame.remainingPins()).toEqual(10); //count pins knocked down?//
```

Comment:
Don't leave comments just lying around. They can get very confusing and make everything quite cluttered